runtime/jcl/common/criu.cpp

```diff
@@ -62,14 +62,7 @@ Java_openj9_internal_criu_InternalCRIUSupport_isCheckpointAllowedImpl(JNIEnv *en
 jboolean JNICALL
 Java_openj9_internal_criu_InternalCRIUSupport_isCRIUSupportEnabledImpl(JNIEnv *env, jclass unused)
 {
-	J9VMThread *currentThread = (J9VMThread *)env;
-	jboolean res = JNI_FALSE;
-
-	if (currentThread->javaVM->internalVMFunctions->isCRIUSupportEnabled(currentThread)) {
-		res = JNI_TRUE;
-	}
-
-	return res;
+	return J9_ARE_ANY_BITS_SET(((J9VMThread*)env)->javaVM->checkpointState.flags, J9VM_CRIU_IS_CHECKPOINT_ENABLED);
```

Comment:
I have a preference for the previous code because it makes it easier to refactor.

Reply:
Restored, also added `CRIUHelpers:: isCRIUSupportEnabled` back along with the new `isCRaCorCRIUSupportEnabled`.